### PR TITLE
Fixes Janicart keys to be more appropriate

### PR DESCRIPTION
**Pull request by Kaedwuff**
The keys to the janicart looked like they had been taken directly from a college fraternity, and didn't really feel appropriate for a corporate setting.

So I replaced them with a set of keys that has a cute little mop on it, which fits the job and also isn't so obnoxious I don't want to use the cart every round for shame of being associated with term 'pussy wagon'

I also cleaned up some of the lazy code and designations, like how there are three keys, and the janitor key icon was just called 'key'.  If we add further keys in the future, perhaps give them more descriptive icon names from the outset.

4259d04be7b214035fd60676357872d6

### CODE REVIEW

**Comment by Kaedwuff** [comment]
Why is this merging off of master... I branched it off of development. @_@

**Comment by ParadoxSpace** [comment]
'Fixes Janicart to be not fun'

**Comment by Kaedwuff** [comment]
Make your own fun instead of relying on tired memes you likely will barely notice are gone after a few weeks.

**Comment by Pacmandevil** [comment]
>and didn't really feel appropriate for a corporate setting.

There's plenty of shit that doesn't feel appropriate for a corporate setting, like giant ass cyan Mohawks, or ballerina IPC HOSs.

I'd Argue that the pussy wagon makes perfect sense, actually. I wouldn't put it past it Icly being a running joke in the janitorial staff, either.

**Comment by Kaedwuff** [comment]
And those also get addressed when they come up.  It's largely an irrelevant comparison, too, because those are player-created content issues, not setting environment issues.  You can call your cart a pussywagon ICly if you want, but there is literally no reason the game should force the meme on people who aren't interested in it.

**Comment by Pacmandevil** [comment]
if you're disgusted by a sprite in 2d spess you have some issues.

regardless -  a less than 200% serious keychain is perfectly able to fit in the setting. Hell when I carried boxes for a living some dude had [one of these](https://www.amazon.com/Neca-porte-cl%C3%A9s-m%C3%A9tal-Pussy-Wagon/dp/B0006FUAE0) on a forklift, and I can perfectly envision some janitor buying one for the cart they work with on station.

If you get disgusted by a good natured joke in 2d spessmens, it's not your responsibility to go on a crusade to remove something you personally don't like for no reason other than "I don't like this". Which is pretty well the only actual reason you've provided, as it makes perfect sense in setting for Nanotrasen to not give a singular flying shit about a keychain of all things.


And seriously, if the phrase "Pussy wagon" makes you uncomfortable, you perhaps need to toughen up a bit. literally all it is is a harmless kill bill reference, and I'd say a fitting one at that.

**Comment by HeyBanditoz** [comment]
Why does Aurora always open some of the strangest PRs in ss13?

**Comment by Pacmandevil** [comment]
because DM is sadly easy to learn so we get stuff like this

**Comment by Pacmandevil** [comment]
> And it makes perfect sense to me that the keys issued to janitorial staff would not have a gag keychain on them.

It also makes perfect sense otherwise, as I'm sure most Janitorial staff wouldn't give a singular shit about it, and I'm sure most of them wouldn't start quivering and have to throw it away before something leaks.

> I don't see why you're complaining about a throwaway pop culture reference with no topical significance.

Isn't that what you're doing here? Literally all this PR is is "I have the social skills expected of someone with a furry avatar, so I can't handle a pop culture reference without feeling uncomfortable. better remove it."

**Comment by skull132** [comment]
Poking in to say hi.

@Pacmandevil that last paragraph was great and all but please keep discussion pertaining to the pull request itself. If you have nothing more to add on that front, then simply don't say anything. I hope that's clear enough.

The last bit applies to everyone viewing this.

**Comment by optimumtact** [comment]
\>furry
\>disgusted by references to pussy to the point of throwing a key down a drain

:thinking:

**Comment by Fire-and-Glory** [comment]
If the whole thing is really so awful and divisive then one could just stick in some "remove keychain" button to separate the two and keep it as-is as the default.

**Comment by Ratvar** [comment]
Make janitor choose his key the same way chaplain does it with nullrod, done. Choice for everyone who's disgusted or not, hooray.

**Comment by Iamgoofball** [comment]
@Kaedwuff 
Have you never seen Kill Bill???'

chapter_2_pussy_wagon

**Comment by Iamgoofball** [comment]
additionally, this joke has been in the game for about 10 years now

come on, you gotta learn to take a joke

**Comment by HeyBanditoz** [comment]
My eyes burn get that pussy outta here

On May 15, 2018 3:17:48 PM MDT, Iamgoofball <notifications@github.com> wrote:
>@Kaedwuff 
>Have you never seen Kill Bill???'
>
>chapter_2_pussy_wagon
>
>
>
>-- 
>You are receiving this because you commented.
>Reply to this email directly or view it on GitHub:
>https://github.com/Aurorastation/Aurora.3/pull/4713#issuecomment-389316613

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by Iamgoofball** [comment]
like seriously the actual character those keys belonged to was some janitor asshole in the hospital the Bride was in

it makes perfect sense for the janitor to have the pussy wagon keys

**Comment by Kaedwuff** [comment]
As I said before I have not seen Kill Bill, and like I said also, I'm going to code it in option to pick what you want your keys to look like.

Everyone should be happy with this, the meme will stay in the game but can chosen to be opted out of.

There is no further need to remind me that it's a reference and complain about me destroying long-standing memes.

**Comment by OneOneThreeEight** [comment]
There's like, no point to be partaking in this crusade of yours to destroy movie reference memes that people otherwise don't care about. I fail to see how this is remotely offensive to roleplay.

It's one thing for IPCs to be able to use blood magic because that's a plausibility thing, it's another to want to remove things from the game that only subtly references pop culture. SS13 is chock full of this. The HOS greatcoat is a reference to Adam Jensen's attire. "You never asked to be this fashionable", I think that's the flavor text for it.

Yes, keys labelled 'pussy wagon' is obviously not the funniest reference to the movie Kill Bill, but it's otherwise harmless and you're the only person as of recent to take issue with this.

In the same vein it's dumb that pacman + paradox felt the need to cal le reddit army xd to brigade this PR, but whatever. Many things are dumb here.

File: code/modules/vehicles/janicart.dm [hunk 1]
```diff
@@ -34,30 +34,30 @@
...
 
-/obj/vehicle/train/cargo/engine/pussywagon/turn_off()
+/obj/vehicle/train/cargo/engine/janicart/turn_off()
 	..()
 
 	if(tow)
-		if(istype(tow,/obj/vehicle/train/cargo/trolley/pussywagon))
-			var/obj/vehicle/train/cargo/trolley/pussywagon/PW = tow
+		if(istype(tow,/obj/vehicle/train/cargo/trolley/janicart))
+			var/obj/vehicle/train/cargo/trolley/janicart/PW = tow
```
Comment (outbools) [inline code/modules/vehicles/janicart.dm]: Those initials translates as Pussy Wagon. Imagine if a coder comes across those filthy initials when trying to improve janicarts and get mad so he decides to abandon codebase, leaving players without features. But that's only a tip of iceberg. Imagine if triggered so much that he will decide to harm himself, did you EVER think about it? Pls do a proper job and remove ALL of those FILTHY, MISOGYNISTIC references.

File: code/modules/vehicles/janicart.dm [hunk 2]
```diff
@@ -34,30 +34,30 @@
...
 			if(PW.mopping)
```
Comment (outbools) [inline code/modules/vehicles/janicart.dm]: REPLACE WITH SOCIETY-FRIENDLY INITIALS

File: code/modules/vehicles/janicart.dm [hunk 3]
```diff
@@ -34,30 +34,30 @@
...
 				PW.mop_toggle()
```
Comment (outbools) [inline code/modules/vehicles/janicart.dm]: WE LIVE IN A SOCIETY
REPLACE WITH SOCIETY-FRIENDLY INITIALS

File: code/modules/vehicles/janicart.dm [hunk 4]
```diff
@@ -34,30 +34,30 @@
...
 			if(PW.hoover)
```
Comment (outbools) [inline code/modules/vehicles/janicart.dm]: REPLACE WITH SOCIETY-FRIENDLY INITIALS
gosh im SHAKING RIGHT NOW
HOW DARE YOU NOT REPLACE THAT?
Do you even care?

File: code/modules/vehicles/janicart.dm [hunk 5]
```diff
@@ -34,30 +34,30 @@
...
 				PW.hoover_toggle()
```
Comment (outbools) [inline code/modules/vehicles/janicart.dm]: Oh. My. God.
REPLACE WITH SOCIETY-FRIENDLY INITIALS
NOW

File: code/modules/vehicles/janicart.dm [hunk 6]
```diff
@@ -66,14 +66,14 @@
 		return
 
 	if(on && tow)
-		if(istype(tow,/obj/vehicle/train/cargo/trolley/pussywagon))
-			var/obj/vehicle/train/cargo/trolley/pussywagon/PW = tow
+		if(istype(tow,/obj/vehicle/train/cargo/trolley/janicart))
+			var/obj/vehicle/train/cargo/trolley/janicart/PW = tow
```
Comment (outbools) [inline code/modules/vehicles/janicart.dm]: REPLACE WITH SOCIETY-FRIENDLY INITIALS
REPLACE WITH SOCIETY-FRIENDLY INITIALS
REPLACE WITH SOCIETY-FRIENDLY INITIALS
REPLACE WITH SOCIETY-FRIENDLY INITIALS
REPLACE WITH SOCIETY-FRIENDLY INITIALS
REPLACE WITH SOCIETY-FRIENDLY INITIALS
REPLACE WITH SOCIETY-FRIENDLY INITIALS
REPLACE WITH SOCIETY-FRIENDLY INITIALS

File: code/modules/vehicles/janicart.dm [hunk 7]
```diff
@@ -82,11 +82,11 @@
...
```
Comment (outbools) [inline code/modules/vehicles/janicart.dm]: statement repeated 8 times gets validated by itself
REPLACE WITH SOCIETY-FRIENDLY INITIALS

**Comment by skull132** [comment]
Apparently a specific crowd are unable to remain civil. O'well.

Anyways. I don't really see a merit to this PR one way or another. The keys are a reference and give personality to the janitor role. If you don't like it, toss the keys down the drain and _roleplay_ it out.

**Pull request closed without merging**